4week/src/pages/CommentPage.tsx

```
@@ -0,0 +1,135 @@
+    data: comments,
+    isLoading: commentsLoading,
+    isFetching: commentsFetching,
+    hasNextPage: commentsHasNextPage,
+    fetchNextPage: commentsFetchNextPage,
+  } = useGetInfiniteComment(numericLpId, 10, order);
+
+  const { ref, inView } = useInView({ threshold: 0 });
+
+  useEffect(() => {
+    if (inView) {
+      !commentsFetching && commentsHasNextPage && commentsFetchNextPage();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)